App.js

```diff
@@ -21,6 +23,7 @@ export default function App() {
           headerShown: false, // Hide header for all screens
         }}
       >
+        <Stack.Screen name="Score" component={ScoreScreen}/>
```

Comment:
`ScoreScreen` should not be the first screen of the app, you can move this line to the last.